src/vma/sock/sock-redirect.cpp

```diff
@@ -724,6 +763,10 @@ int bind(int __fd, const struct sockaddr *__addr, socklen_t __addrlen)
 extern "C"
 int connect(int __fd, const struct sockaddr *__to, socklen_t __tolen)
 {
+	BULLSEYE_EXCLUDE_BLOCK_START
```

Comment:
@rafiw Here you are calling get_orig_funcs() at function startup. Which is OK since connect() is not on the fast path. However, please look below at setsockopt()...